STLYEGUIDE_SPRESS_UPDATE/src/content/elements/element-buttons.html

```diff
@@ -0,0 +1,15 @@
+---
+layout: element
+section: element
+
+title: Buttons
+description: Create basic button styles for the site. This includes only the button element and the inputs for submit and reset.
```

Comment:
@aCyborg I'd drop "Create" so that this doesn't sound like instructions for the client. Also drop "only," since it implies that there are other button-like things we are leaving out.